swift/app.py

```diff
@@ -35,7 +44,9 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         """
         super().__init__(parent=parent)
         self.name = name
+        self.swiftcall = SwiftcallProxy(self.swiftcallRequested)
         self.received.connect(self._receivedMessage)
+        self.swiftcallReturned.connect(self._receivedResult)
```

Comment:
How about including 'swiftcall' in the name of slot?
So far, we included 'swiftcall' in names about `swiftcall`, such as `swiftcallReqeusted`, `swiftcall`, etc.

Reply:
Good idea! I will change the name.